src/app/containers/FeaturesAnalysis/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,293 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CpsRelatedContent should render Story Feature components in Live environment 1`] = `
+<DocumentFragment>
+  <section
+    aria-labelledby="features-analysis-heading"
+    class="GhostGrid-sc-12lwanc-1 kGasFS"
+    role="region"
+  >
+    <div
+      class="GridItemConstrainedLarge-sc-12lwanc-5 Wrapper-r0jyv5-0 ftAujp"
+    >
+      <div
+        class="GridItemConstrainedLarge-sc-12lwanc-5 Wrapper-r0jyv5-0 ftAujp"
+      >
+        <div
+          class="SectionLabelWrapper-sc-1x4gwjz-1 gKFUvc StyledSectionLabel-r0jyv5-1 dRpDSS"
+        >
+          <div
+            class="Bar-sc-1x4gwjz-0 jsgdXU"
+          />
+          <h2
+            class="Heading-sc-1x4gwjz-2 eZofTG"
+          >
+            <div
+              class="FlexColumn-wx5kdp-0 dgpsiS"
+            >
+              <span
+                class="FlexRow-wx5kdp-2 dVBNyZ"
+              >
+                <span
+                  class="Title-wx5kdp-4 jiurmA"
+                  dir="ltr"
+                  id="features-analysis-heading"
+                >
+                  Features Analysis 
+                </span>
+              </span>
+            </div>
+          </h2>
+        </div>
+        <ul
+          class="StoryPromoUl-sc-171lqjd-1 isCYWc"
+          role="list"
+        >
+          <li
+            class="StoryPromoLi-sc-171lqjd-0 dqJaJG"
+            role="listitem"
+          >
+            <div
+              class="StoryPromoWrapper-sc-1dvfmi3-0 buBJys"
+            >
+              <div
+                class="ImageGridItem-sc-1xk56cj-0 fNYKlW"
+                dir="ltr"
+              >
+                <div
+                  class="ImageContentsWrapper-sc-1dvfmi3-1 cinwNl"
+                >
+                  <div
+                    class="ImagePlaceholder-sc-11u25v2-0 fNqlUz"
+                  >
+                    <div
+                      class="lazyload-placeholder"
+                    />
+                  </div>
+                  <div
+                    class="ImageOverlayWrapper-sc-1dvfmi3-2 dqBekx"
+                  />
+                </div>
+              </div>
+              <div
+                class="TextGridItem-sc-1lkbq5i-0 jJWtYv"
+                dir="ltr"
+              >
+                <h3
```

Comment:
Just realised that we shouldn't have headings (h3's) for each of the promos, as there is no related content below them and this fails an accessibility standard. Is it possible to change this to a div please?

Reply:
Yes, this has been overridden to make it a DIV.

Reply:
@greenc05 the use of h3 appears to be an existing problem with the Related Content component which the work here is heavily based on (e.g. see https://www.bbc.com/pidgin/tori-51728785).

We've just had a discussion and are going to create a separate issue to look at an accessible solution for both Related Content and Features and Analysis in the round, rather than applying an overly-specific fix here.

Reply:
See https://github.com/bbc/simorgh/issues/5793

Reply:
Ok, so apologies if my use of the words 'related content' confused things here. If there is no text content below the promo title, then the title should not be a heading, as headings should have related content below them. So in the case of F&A, each li would not have any content below the 'title'. However in the 'Related content' component we have the time stamp below the promo title/heading so this can be a heading https://github.com/bbc/bbc-a11y/blob/master/lib/standards/tests/contentMustFollowHeadings.js

Reply:
The only other exception that I'm aware of is the onwards journey component in the middle of the article/story body text, this component cannot have any headings at all due to being in the article body and would head up all the article/story content below it, which would not be accurate